src/site/index.html

```
@@ -445,93 +555,109 @@ <h3 class="font-bold text-lg mb-6" data-segment-id="index-div-h3-78">Located at<
+                const key = img.getAttribute('data-media-key');
+                const m = media[key];
+                if (m?.url) {
+                    img.src = m.url;
+                    if (m.alt_text) img.alt = m.alt_text;
+                }
+            });
+        })();
+    </script>
+
+
+    </section>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </section> ]

(tag-pair)